src/components/image_upload.tsx

```diff
@@ -214,12 +228,26 @@ export class ImageUpload extends React.Component<ImageUploadProps, ImageUploadSt
         );
     }
 
+    private _renderFileSizeExceedsMessage(): JSX.Element {
```

Comment:
Why an extra method? You can just put this in `render()` below.

Reply:
The `<InputFieldValidation />` components only has a few lines so that is fine to put in `render()`. However I would prefer to put components that have more lines (complex logic/functions etc.) in functions...